From 944fa2afca2bd6bd4d1d5aecd265fd4756ee44e2 Mon Sep 17 00:00:00 2001
From: Li kunyu <kunyu@nfschina.com>
Date: Fri, 18 Nov 2022 16:10:24 +0900
Subject: [PATCH] boot/measure: fix oom check

(cherry picked from commit fc0cc6db1ecbaa16513125d3fd1a7d11e391a8ee)

Related: #2138081
---
 src/boot/measure.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/boot/measure.c b/src/boot/measure.c
index 4f16acedf0..0bbd386449 100644
--- a/src/boot/measure.c
+++ b/src/boot/measure.c
@@ -897,7 +897,7 @@ static int verb_sign(int argc, char *argv[], void *userdata) {
                         }
 
                         _cleanup_free_ void *sig = malloc(ss);
-                        if (!ss) {
+                        if (!sig) {
                                 r = log_oom();
                                 goto finish;
                         }
